Editor/EditorUtils.cs

```diff
@@ -96,6 +96,15 @@ internal static string GetCoreVersion()
             return "unknown";
         }
 
+        /// <summary>
+        /// Returns a list of scriptable objects from provided type;
+        /// </summary>
+        internal static IEnumerable<T> GetAllScriptableObjects<T>() where T : ScriptableObject
```

Comment:
Does it work with streaming assets? Can streaming assets be scriptable objects?

Reply:
Good question, I'll check it out 🤔

Reply:
No, it can't.
And even if it could, it is editor only which could be a problem 😕